OpenCL_C.txt

```diff
@@ -2208,9 +2207,9 @@ only for the lifetime of the work-group executing the kernel.
 --
 
 The `+__constant+` or `constant` address space name is used to describe
-variables accessible globally (declared in program scope or inside functions
-with `static`/`extern` storage class specifier) as read-only variables.
-These read-only variables can be accessed by all work-items of different
+read-only variables that are accessible globally. They may
+be declared in program scope or inside functions with a `static` or `extern`
+storage class specifier, and can be accessed by all work-items of different
 kernels during their execution.
```

Comment:
```suggestion
 can be accessed by all work-items or by different kernels during the program execution.
```

Reply:
I realized that the original wording might have been confusing since it might have implied that the function scope variables can be accessed by different kernels which is not possible.